docs/proposal-rose-suite-run.md

```diff
@@ -265,8 +265,9 @@ Proposed contents of the `log` directory in the run directory.
 * `db` - no change
 * `job/` - no change
 * `scheduler/` - replaces `suite/`
-* `conf/` - replaces `suiterc/` and `rose-conf/*.conf`
+* `conf/` - replaces `suiterc/` and `rose-conf/*.conf` (contains Cylc + Rose confs)
   * see also [cylc-flow #3763](https://github.com/cylc/cylc-flow/issues/3763)
-* `install/<timestamp>-install.log` (replaces `rose-suite-run.log`)
-* `install/<timestamp>-reload.log` (currently appended to `rose-suite-run.log`)
-* `install/<timestamp>-vc.log` (replaces rose-conf/*.version)
+* `-install/` - installation logs
+  * `install/<timestamp>-install.log` (replaces `rose-suite-run.log`)
+  * `install/<timestamp>-install.log` (currently the reload log is appended to `rose-suite-run.log`, Rose is no longer responsible for calling reload)
```

Comment:
This line is a bit confusing because it repeats the filename from the line above and I'm not sure the comments about reload are needed. Just remove this line entirely?

Reply:
(comment was meant to be against line 272)